lib/SILOptimizer/Utils/Generics.cpp

```diff
@@ -145,7 +145,7 @@ static bool cacheSpecialization(SILModule &M, SILFunction *F) {
 
   if (M.getOptions().Optimization >= SILOptions::SILOptMode::Optimize &&
       F->getLinkage() != SILLinkage::Public &&
-      F->getModule().getSwiftModule()->getName().str() == STDLIB_NAME) {
+      F->getModule().getSwiftModule()->getName().str() == "SwiftOnoneSupport") {
```

Comment:
Probably this magic module name should be factored into a constant like `STDLIB_NAME`.